### ISSUE TITLE
cfg训练的负样本条件y的设置

### ISSUE THREAD

**Issue by YangHui90** [TongTong313/rectified-flow#3]
感谢分享代码和相关视频教程。

看到在train.py脚本里在训练条件模型时，y = torch.cat([y, -torch.ones_like(y)], dim=0)这行代码把负样本条件初始化为-1了，可以初始化成其他值么？然后我看到infer.py里cfg推理时，负样本的条件是None。不太理解这么做的原因是什么？为啥推理代码里不初始化为-1？另外，如果推理代码里负样本条件不需要初始化某个值，训练阶段的的负样本条件是不是可以随意初始化（只要与正样本条件不重合即可）？

盼答疑解惑，谢谢。

**Comment by TongTong313** [comment]
首先，你可以随意初始化，只要不和正样本条件重合，但我觉得效果未必会好，因为不重合的情况有无数种，你无法都训练到位。其次，-1是我设置的，你也可以随意别的值，你在推理的时候当然也可以把y设置为-1表示无条件，这并不冲突。最后，为什么设置为None也可以，建议看一下我B站的代码讲解视频都有讲，记住每一集都要看哦~
